src/index.test.ts

```diff
@@ -417,6 +417,254 @@ describe('Output Plugin', () => {
+      // Change fetchSite's return value to include a custom dcat config
+      const customConfigSiteModel: IModel = _.cloneDeep(mockSiteModel) as any;
+      (customConfigSiteModel.data || {}).feeds = {
+        dcatUS11: {
+          "title": "{{name}}",
+          "description": "{{description}}",
+          "keyword": "{{tags}}",
+          "issued": "{{created:toISO}}",
+          "modified": "{{modified:toISO}}",
+          "publisher": { "name": "{{source}}" },
+          "contactPoint": {
+            "fn": "{{owner}}",
+            "hasEmail": "{{orgContactEmail}}"
+          },
+          "landingPage": "some silly standard",
+        }
+      }
+      mockFetchSite.mockResolvedValue(customConfigSiteModel);
+      [plugin, app] = buildPluginAndApp();
+
+      // A hack to extract the request object that is actually served
+      let servedRequest: any;
+      app._router.stack.pop();
+      app.get('/dcat', (async (req, res) => {
+        servedRequest = req;
+        await plugin.serve.bind(plugin)(req, res);
+      }));
+
+      await request(app)
+        .get('/dcat')
+        .set('host', siteHostName)
+        .query({ sort: 'Date Modified|modified' })
+        .expect('Content-Type', /application\/json/)
+        .expect(200)
+        .expect(() => {
+          expect(mockGetDataStreamDcatUs11).toHaveBeenCalledWith(siteHostName, customConfigSiteModel, customConfigSiteModel.data?.feeds.dcatUS11);
+          expect(servedRequest.res.locals.searchRequest.options.sortField).toEqual('modified');
+          expect(servedRequest.res.locals.searchRequest.options.sortOrder).toEqual('desc');
+        });
+    });
+
+    it('Constructs a search request using a sort of three input components', async () => {
+      // Change fetchSite's return value to include a custom dcat config
+      const customConfigSiteModel: IModel = _.cloneDeep(mockSiteModel) as any;
+      (customConfigSiteModel.data || {}).feeds = {
+        dcatUS11: {
+          "title": "{{name}}",
+          "description": "{{description}}",
+          "keyword": "{{tags}}",
+          "issued": "{{created:toISO}}",
+          "modified": "{{modified:toISO}}",
+          "publisher": { "name": "{{source}}" },
+          "contactPoint": {
+            "fn": "{{owner}}",
+            "hasEmail": "{{orgContactEmail}}"
+          },
+          "landingPage": "some silly standard",
+        }
+      }
+      mockFetchSite.mockResolvedValue(customConfigSiteModel);
+      [plugin, app] = buildPluginAndApp();
+
+      // A hack to extract the request object that is actually served
+      let servedRequest: any;
+      app._router.stack.pop();
+      app.get('/dcat', (async (req, res) => {
+        servedRequest = req;
+        await plugin.serve.bind(plugin)(req, res);
+      }));
+
+      await request(app)
+        .get('/dcat')
+        .set('host', siteHostName)
+        .query({ sort: 'Date Modified|modified|asc' })
```

Comment:
We're good for now, but I've been dealing with this sort serialization in the gallery and I question the utility of having the english name be a part of the string. For example, in the gallery we ignore the english name entirely. We just look up the field name and order. 

I think you're good for this PR, just something I'm considering dropping for the future.